views/templates/front/expressCheckout.tpl

```diff
@@ -46,43 +46,40 @@
   }
   #paypal-button-container {
     max-width:300px;
+  }
+  .page-cart #paypal-button-container {
     margin: auto;
   }
 </style>
 
 <script>
-/**
- * Load paypal script
- */
-function loadPaypalScript() {
-  if (typeof paypalSdkPsCheckout !== 'undefined') {
-    return;
-  }
-
-  let psCheckoutScript = document.getElementById('psCheckoutPaypalSdk');
+  const checkoutLink = "{$checkoutLink|escape:'javascript':'UTF-8' nofilter}";
```

Comment:
This should be `|escape:'javascript':'UTF-8'` or `nofilter`, but both do not make sense.

Reply:
It's required because on 1.7 PrestaShop apply escaping + apply escape

Reply:
Try with and without ;)